### ISSUE TITLE
[Feature Request]  Fast Forward without speeding up sound

### ISSUE THREAD

**Issue by Woedenaz** [libretro/RetroArch#14529]
## Description

This is pretty niche request, I understand, but I thought I'd ask anyway in the off-chance others might be interested in it.

Would be nice if there was a way to allow for fast forwarding that also keeps the sound from being sped up. I was doing some googling and [there was another person that asked for this on Reddit some months ago.](https://www.reddit.com/r/RetroArch/comments/utulzv/how_to_make_the_audio_on_normal_speed_while_fast/) Seems they chose not to make the request so I'm doing so instead. :]

Another user in that thread mentioned it _might_ be possible using a similar technique as Run-Ahead but I do not know enough to say if that is accurate or not.

**Comment by SkyOfSteel** [comment]
Bump, cause I'd love to have this feature, too.

**Comment by sonninnos** [comment]
It will never be possible.

**Comment by SkyOfSteel** [comment]
I mean, Cheat Engine does it somehow, and there are speculations in the linked thread about how it could be possible (by emulating the game twice in the background and splitting audio and gameplay this way), so I'll hold on to my hope.

**Comment by sonninnos** [comment]
No. Pure speculation by a non-coder. Emulation simply can't work that way universally.

**Comment by lonkelle** [comment]
if, when the user turned on fast forward a save state was created and another thread opened up the state without inputs just to play the music it would work

you'd lose all sound effects and possibly here incorrect ones. the save state would need to synchronize with your current state every now and then but I can't think of an intelligent way to make that synchronization 

so basically it's possible just like run ahead is but with some huge caveats that would leave sound effects out and music getting desynced when screens change and stuff
